src/data-tutorials/users.tsx

```diff
@@ -845,6 +845,13 @@ const Users: User[] = [
     website: 'https://medium.com/celodevelopers/introduction-to-creating-nfts-on-celo-eb7240a71cc0',
```

Comment:
kindly change this directory.

Reply:
And this too, because that website is not included in my tutorial and I'm seeing stuffs about nfts again

Reply:
Kindly make changes, commit changes and make a PR. Once you're done, kindly let me know. I will proceed this to the next steps

Reply:
This directory is not on my tutorial